x509util/certcheck/certcheck.go

```diff
@@ -34,6 +34,7 @@ var (
 	intermediate             = flag.String("intermediate", "", "Intermediate CA certificate file")
 	useSystemRoots           = flag.Bool("system_roots", false, "Use system roots")
 	verbose                  = flag.Bool("verbose", false, "Verbose output")
+	failNFE                  = flag.Bool("fail_nfe", true, "Set non-zero exit code for non-fatal errors in parsing")
```

Comment:
This would more commonly be called --strict I think.

Reply:
Yep, that's a much better name; done.